src/rocprof_compute_profile/profiler_base.py

```diff
@@ -77,6 +78,22 @@ def join_prof(self, out=None):
                 out = self.__args.path + "/pmc_perf.csv"
             files = glob.glob(self.__args.path + "/" + "pmc_perf_*.csv")
             files.extend(glob.glob(self.__args.path + "/" + "SQ_*.csv"))
+
+            if self.get_args().hip_trace:
+                # remove hip api trace ouputs from this list
+                files = [
+                    f
+                    for f in files
+                    if not fnmatch.fnmatch(os.path.basename(f), "*_hip_api_trace.csv")
```

Comment:
Not a big deal...why fnmatch but not regex?
If there is portable solution, we should to go for it now in case WIN support in the future:)